src/IBM.Watson.Assistant.v2/Model/RuntimeResponseGeneric.cs

```diff
@@ -165,6 +165,11 @@ public class PreferenceEnumValue
         [JsonProperty("agent_unavailable", NullValueHandling = NullValueHandling.Ignore)]
         public AgentAvailabilityMessage AgentUnavailable { get; protected set; }
         /// <summary>
+        /// Routing or other contextual information to be used by target service desk systems.
+        /// </summary>
+        [JsonProperty("transfer_info", NullValueHandling = NullValueHandling.Ignore)]
+        public DialogNodeOutputConnectToAgentTransferInfo TransferInfo { get; protected set; }
+        /// <summary>
```

Comment:
TransferInfo was one of the issues within .net and java. I would double check this if this is bringing back that problem.